version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "1.3.5-SNAPSHOT"
+version in ThisBuild := "1.3.6-SNAPSHOT"
```

Comment:
No need to update this, this is incremented automatically when a new release is created